src/model/Model.ts

```diff
@@ -194,6 +194,12 @@ export class Model {
    * Get the store instance.
    */
   get $store(): Store<any> {
+    assert(this._store !== undefined, [
+      'The store instance is not injected into the model instance. You might',
+      'be trying to instantiate the model directly. Please use',
+      '`repository.make` method to create a new model instance.'
+    ])
```

Comment:
**_The store instance..._** implies a singleton, could this be **_A Vuex Store instance..._**?

Reply:
Ah good point. Yes let's make it "A" Vuex Store instance 👍  I'll fix this and merge it.